doc/manual.asciidoc

```diff
@@ -821,6 +821,9 @@ behaviors:
 `$` followed by a newline:: escape the newline (continue the current line
 across a line break).
 
+`$|`:: a newline. Can be used to put literal newlines into ninja variables
```

Comment:
I think this belongs between `$:` and `$$` below, rather than here.  To be consistent with the other escapes, maybe something like:

`$|`:: a newline.  (This becomes a literal newline in the variable assignment.)